ga/20.0.0.7/full/Dockerfile.ubuntu.ibmjava8

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM websphere-liberty:kernel
+FROM websphere-liberty:20.0.0.7-kernel
```

Comment:
I believe this needs to be the fully quantified kernel tag, `20.0.0.7-kernel-java8-ibmjava`